docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -104,16 +113,16 @@ error | When this style is not being followed, show a compiler error.
 dotnet_style_qualification_for_field = false:suggestion
 ```
 
-### <a name="this_and_me_properties">Properties</a>
+### <a name="this_and_me_properties">Properties (IDE0003/IDE0009) </a>
 
 |  Option Name | `dotnet_style_qualification_for_property` |
 | ------------- |:-------------:|
 | **Applicable Languages** | C# and Visual Basic

```

Comment:
Whatever the coded default value for the setting is. For example, `dotnet_style_qualification_for_property` is defined with a default value here:
https://github.com/dotnet/roslyn/blob/9e67e47234a7ea168168300c2ae9c02f29826288/src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs#L32-L35